### PR TITLE
Added post about auth0-ember-simple-auth

### PR DESCRIPTION
**Pull request by benschwarz**

### CODE REVIEW

**Comment by vctrfrnndz** [comment]
:boom:

File: _posts/2015-06-26-auth0-ember-simple-auth.markdown [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+- ember
+- authentication
+- plugin
+- library
+- open source
+
+---
+At Auth0, we’re huge fans of [Ember](http://emberjs.com), the Javascript framework made famous for writing ambitious single page applications (also known as SPA).
+
+Unfortunately, up until now our support and resources for Ember have not equalled our enthusiasm for Ember. Thankfully, starting today, we have some great news —
```
Comment (mgonto) [inline _posts/2015-06-26-auth0-ember-simple-auth.markdown]: `for it`

File: _posts/2015-06-26-auth0-ember-simple-auth.markdown [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+
+![Auth0 + Ember logos](http://cdn.auth0.com/blog/ember-simple-auth/auth0-ember.png)
+
+# Introducing an official 'ember simple auth' addon
+
+For those familiar with Ember, it might go without saying that “[ember-simple-auth](http://ember-simple-auth.com)” is the most popular authentication helper library around. It implements a strategy based plugin system that allows users to write their own custom authentication strategies.
+
+That is exactly what one of our customers, [Aram Zadikian](https://github.com/brancusi) did, and thanks to his generosity, we’re able to share his fantastic work with you today!
```
Comment (mgonto) [inline _posts/2015-06-26-auth0-ember-simple-auth.markdown]: This is a bit of an over reach I think :D. Thoughts?

File: _posts/2015-06-26-auth0-ember-simple-auth.markdown [hunk 3]
```diff
@@ -0,0 +1,86 @@
...
+
+tags:
+- ember
+- authentication
+- plugin
+- library
+- open source
+
+---
+At Auth0, we’re huge fans of [Ember](http://emberjs.com), the Javascript framework made famous for writing ambitious single page applications (also known as SPA).
```
Comment (mgonto) [inline _posts/2015-06-26-auth0-ember-simple-auth.markdown]: Add a TL;DR with the link to the repository and the example

**Comment by mgonto** [comment]
Is the docs page ready? We shouldn't ship this until the docs page is finished and Ember shows up in the docs with the correct tutorial.

Also @rickyrauch wanted to change some images I think.

File: _posts/2015-06-26-auth0-ember-simple-auth.markdown (hunk 2 above)
Comment (benschwarz) [inline _posts/2015-06-26-auth0-ember-simple-auth.markdown]: I'm not sure if this is too far @mgonto — Its great to encourage customers to reach out and make contributions just like this one. Perhaps "generosity" is the wrong word though?

**Comment by mgonto** [comment]
2 more comments:
* We need to redo the Giff with the Auth0 badge on top
* We will release this once the doc page is finished and we should point to the doc page

Thanks!

File: _posts/2015-06-26-auth0-ember-simple-auth.markdown (hunk 3 above)
Comment (benschwarz) [inline _posts/2015-06-26-auth0-ember-simple-auth.markdown]: Done

**Comment by benschwarz** [comment]
screen shot 2015-07-07 at 2 43 20 pm

@rickyrauch @woloski 

- [ ] Colour darkened for better contrast
- [ ] Icon changed to ember logo

File: _posts/2015-06-26-auth0-ember-simple-auth.markdown [hunk 4]
```diff
@@ -0,0 +1,90 @@
...
+import ApplicationRouteMixin from 'simple-auth/mixins/application-route-mixin';
+
+export default Ember.Route.extend(ApplicationRouteMixin, {
+  actions: {
+    sessionRequiresAuthentication: function(){
+      // Check out the docs for all the options:
+      // https://auth0.com/docs/libraries/lock/customization
+
+      // These options will request a refresh token and launch lock.js in popup mode
+      var lockOptions = {authParams:{scope: 'openid offline_access'}};
```
Comment (mgonto) [inline _posts/2015-06-26-auth0-ember-simple-auth.markdown]: Remove offline_access from here and we can publish :).

**Pull request merged**